addons/binding/org.openhab.binding.squeezebox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,243 @@
+		<label>Control</label>
+		<description>Control the Zone Player, e.g. start/stop/next/previous/ffward/rewind</description>
+		<category>Player</category>
+	</channel-type>
+	<channel-type id="stream">
+		<item-type>String</item-type>
+		<label>Stream URL</label>
+		<description>Play the given HTTP or file stream (file:// or http://). </description>
+	</channel-type>
+	<channel-type id="sync">
+		<item-type>String</item-type>
+		<label>Sync Player</label>
+		<description>Add another player to your device for synchronized playback (other player mac address)</description>
+	</channel-type>
+	<channel-type id="playListIndex">
+		<item-type>Number</item-type>
+		<label>Playlist Index</label>
+		<description>Playlist index</description>
+	</channel-type>
+	<channel-type id="currentPlayingTime">
+		<item-type>Number</item-type>
+		<label>Current Playing Time</label>
+		<description>Current Playing Time</description>
+	</channel-type>
+	<channel-type id="currentPlaylistShuffle">
+		<item-type>Number</item-type>
+		<label>Shuffle Mode</label>
+		<description>Current playlist shuffle mode</description>
+				<state>
+			<options>
+				<option value="0">No Shuffle</option>
+				<option value="1">Shuffle Songs in PLaylist</option>
+				<option value="2">Shuffle Albums in PLaylist</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="currentPlaylistRepeat">
+		<item-type>Number</item-type>
+		<label>Repeat Mode</label>
+		<description>Current playlist repeat Mode</description>
+		<state>
+			<options>
+				<option value="0">No Repeat</option>
+				<option value="1">Repeat Song</option>
+				<option value="2">Repeat Playlist</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<!-- Squeezebox variables -->
+	<channel-type id="title">
+		<item-type>String</item-type>
+		<label>Title</label>
+		<description>Title of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="remotetitle">
+		<item-type>String</item-type>
+		<label>Remote Title</label>
+		<description>Remote Title (Radio) of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="album">
+		<item-type>String</item-type>
+		<label>Album</label>
+		<description>Album name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="artist">
+		<item-type>String</item-type>
+		<label>Artist</label>
+		<description>Artist name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="year">
+		<item-type>String</item-type>
+		<label>Year</label>
+		<description>Release year of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="genre">
+		<item-type>String</item-type>
+		<label>Genre</label>
+		<description>Genre name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="coverarturl">
```

Comment:
Where does this url point to? To an internet url or to the local squeezebox server?